go/filemanager/pfsmodules/command.go

```diff
@@ -0,0 +1,46 @@
+package pfsmodules
+
+import (
+	"io"
+	"strings"
+
+	log "github.com/golang/glog"
+)
+
+const (
+	// DefaultMultiPartBoundary is the default multipart form boudary
+	DefaultMultiPartBoundary = "8d7b0e5709d756e21e971ff4d9ac3b20"
```

Comment:
I think in general const block can be put together:
```go
const (
    DefaultMultiPartBoundary = "8d7b0e5709d756e21e971ff4d9ac3b20"
    MaxJSONRequestSize = 2048
```

If you want to write them separately, you can do:
```go
const DefaultMultiPartBoundary = "8d7b0e5709d756e21e971ff4d9ac3b20"
const MaxJSONRequestSize = 2048
```